src/snapshotter.js

```diff
@@ -108,6 +128,8 @@ export default class SnapShotter {
 
       if (this._removeSelectors) await this.removeTheSelectors();
 
+      if (this._onReadyScript) await this.executeScript(this._onReadyScript);
```

Comment:
should this be after `removeSelectors`? 

I can think of arguments for both cases. Just not sure what one would be more appropriate, I feel like it should be before we remove selectors.
I call upon the all-knowing @shendriksen for secondary input

Reply:
I think it's good to put it here as it is when we are in a 'ready' state to perform the snapshotting.

Reply:
I might actually move this before removeSelectors. Agreed with Matt that there might be some scenario where we do a click event and then remove any dynamic elements afterwards